Contribute/how-to-write-use-markdown.md

```diff
@@ -383,10 +383,329 @@ You can see an example of selectors in action at the [Azure docs](https://docs.m
+
+- `cloudshell-powershell` - Enables the Azure PowerShell Cloud Shell, as in the preceding
+  example
+- `cloudshell-bash` - Enables the Azure Cloud Shell
+- `try-dotnet` - Enables Try .NET
+- `try-dotnet-class` - Enables Try .NET with class scaffolding
+- `try-dotnet-method` - Enables Try .NET with method scaffolding
+
+For the Azure Cloud Shell and PowerShell Cloud Shell, users can run commands against only their own
+Azure account.
+
+[Try .NET](https://github.com/dotnet/try) enables interactive execution of .NET code (C#) in the browser. For Try .NET, there are three options for interactivity: `try-dotnet`, `try-dotnet-class`, and `try-dotnet-method`. Use of these options require no additional configuration within the code snippet. The namespaces currently available by default are:
+
+- System
+- System.Linq
+- System.Collections.Generic
+- System.Text
+- System.Globalization
+- System.Text.RegularExpressions
+
+The `try-dotnet` attribute value enables users to run C# code in the browser without the need to wrap the code in any custom code.
+
+Example:
+
+```md
+:::code source="relative/path/source.cs" interactive="try-dotnet"::
+```
+
+The `try-dotnet-class` value applies class-level scaffolding to the code passed to the interactive component.
+
+```md
+:::code source="relative/path/source.cs" interactive="try-dotnet-class":::
+```
+
+Example:
+
+Code Snippet without Class Scaffolding Applied
+
+```md
+public static void Main()
+    {  
+        // Specify the data source.  
+        int[] scores = new int[] { 97, 92, 81, 60 };        // Define the query expression.
+
+        IEnumerable<int> scoreQuery =
+            from score in scores  
+            where score > 80  
+            select score;
+
+        // Execute the query.  
+        foreach (int i in scoreQuery)
+        {  
+            Console.Write(i + " ");
+        }
+    }  
+}
+```
+
+Code Snippet with Class Scaffolding Applied
+
+```md
+class NameOfClass {
+
+   public static void Main()
+    {
+        // Specify the data source.
+        int[] scores = new int[] { 97, 92, 81, 60 };
+
+        // Define the query expression.
+        IEnumerable<int> scoreQuery =
+            from score in scores
+            where score > 80
+            select score;
+
+        // Execute the query.
+        foreach (int i in scoreQuery)
+        {
+            Console.Write(i + " ");
+        }
+    }  
+}
+```
+
+The `try-dotnet-method` value applies method-level scaffolding to the code passed to the interactive component.
+
+```md
+:::code source="relative/path/source.cs" interactive="try-dotnet-method"::
+```
+
+Example:
+
+Code Snippet without Method Scaffolding Applied
+
+```md
+/*Print some string in C#*/
+
+Console.WriteLine("Hello C#.);
+
+Console.ReadKey();
+```
+
+Code Snippet with Method Scaffolding Applied
+
+```md
+public static void Main(string args[]) {
+
+/*Print some string in C#*/
+
+Console.WriteLine("Hello C#.);
+
+Console.ReadKey();
```

Comment:
APIs like `ReadKey` are problematic in try.net. This line should be removed from the snippet (or be in the code, but not included in the try.net region).

Reply:
I'll remove from the snippet